apps/web/next.config.js

```diff
@@ -80,6 +82,8 @@ const contentSecurityPolicy = {
   'worker-src': ["'self'"],
   'connect-src': [
     "'self'",
+    '*',
+    'data:',
```

Comment:
I think you don't need this anymore, since we got custom CSP just for username profile